### ISSUE TITLE
Refactor TVTK Parser

### ISSUE THREAD

**Issue by dmsurti** [enthought/mayavi#204]
Currently the TVTK parser code handles all the VTK versions we support:

* 5.10
* 6.x

The code needs to be refactored into separate modules for each VTK version. This will affect the following modules:

* `code_gen`
* `wrapper_gen`
* `vtk_module`

**Comment by itziakos** [comment]
I agree, I might give it a go and make a proposal. I also think that we need to increase test coverage before we move forward with such refactoring.
